src/types.ts

```diff
@@ -74,6 +74,20 @@ export interface RetailCalendar {
   numberOfWeeks: number
   months: RetailCalendarMonth[]
   weeks: RetailCalendarWeek[]
+  days: RetailCalendarDay[]
+}
+
+export interface RetailCalendarDay {
+  weekOfYear: number;
+  monthOfYear: number;
+  dayOfWeek: number // 1 = Monday, 7 = Sunday
```

Comment:
will this be 1th index similar to month?

Reply:
I just updated the comment. Now 1 = first day of week, 7 = last day of week. So it doesn't correspond to Mon-Fri.